### PR TITLE
Adjust the link to the right troubleshooting doc page

### PR DESCRIPTION
**Pull request by ghost**
see #3366

https://github.com/kubernetes/kubernetes.github.io/issues/3366

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,6 @@ That said, if you have questions, reach out to us
 [Scalable Microservices with Kubernetes]: https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615
 [Submit Queue]: http://submit-queue.k8s.io/#/e2e
 [Submit Queue Widget]: http://submit-queue.k8s.io/health.svg?v=1
-[troubleshooting guide]: http://kubernetes.io/docs/troubleshooting
+[troubleshooting guide]: https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/
```
Comment (xiangpengzhao) [inline README.md]: Yeah, according to https://github.com/kubernetes/kubernetes.github.io/pull/3337 , this change is right.

**Review by xiangpengzhao — APPROVED** [review]

**Comment by grodrigues3** [comment]
@k8s-bot ok to test

**Pull request merged**